### PR TITLE
split delimiters so they aren't present in an unmodified binary

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: pkg/embed/embed.go [hunk 1]
```diff
@@ -57,15 +58,15 @@ func EmbedReleaseDataInBinaryReader(binReader io.Reader, binSize int64, releaseD
 	encodedRelease := base64.StdEncoding.EncodeToString(releaseData)
 
 	newBinSize := binSize
-	newBinSize += int64(len(beginReleaseDelimiter))
+	newBinSize += int64(len(beginReleaseDelimiterBytes()))
 	newBinSize += int64(len(encodedRelease))
 	newBinSize += int64(len(endReleaseDelimiter))
```
Comment (sgalsaleh) [inline pkg/embed/embed.go]:
```suggestion
	newBinSize += int64(len(endReleaseDelimiterBytes()))
```

Comment (sgalsaleh) [reply pkg/embed/embed.go]: did tests not catch this?

Comment (laverya) [reply pkg/embed/embed.go]: tests did not catch this

File: pkg/embed/embed_test.go [hunk 2]
```diff
@@ -41,15 +41,18 @@ func TestEmbedReleaseDataInBinary(t *testing.T) {
 	gotBinContent, err := os.ReadFile(outputFile.Name())
 	assert.NoError(t, err)
 
-	wantBinContent := append(binContent, []byte(beginReleaseDelimiter)...)
+	beginString := "-----BEGIN APP RELEASE-----"
```
Comment (sgalsaleh) [inline pkg/embed/embed_test.go]: why not use the `beginReleaseDelimiterBytes()` and `endReleaseDelimiterBytes()` functions here as well?

Comment (laverya) [reply pkg/embed/embed_test.go]: I'll do that and add unit tests for the bytes functions

File: pkg/embed/embed.go (hunk 1 above)
Comment (sgalsaleh) [reply pkg/embed/embed.go]: seems like they should be updated then?

Comment (laverya) [reply pkg/embed/embed.go]: yep yep will do

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**